addons/account/static/src/js/reconciliation/reconciliation_renderer.js

```diff
@@ -654,11 +655,21 @@ var LineRenderer = Widget.extend(FieldManagerMixin, {
     //--------------------------------------------------------------------------
     // Handlers
     //--------------------------------------------------------------------------
+    /**
+     * The event on the partner m2o widget was propagated to the bank statement
+     * line widget, causing it to expand and the others to collapse. This caused
+     * the dropdown to be poorly placed and an unwanted update of this widget.
+     *
+     * @private
+     */
+    _onStopPropagation: function(ev) {
+        ev.stopPropagation();
+    },
 
     /**
      * @private
      * @param {MouseEvent} event
-     */
+     */Explain why you stop it
```

Comment:
:-)

Reply:
you should remove this ^^

Reply:
that will fix the linter test too :+1: 
That way we can definitely get rid of PR

Reply:
I have no idea how I put that there, but its gone now :)